src/types.ts

```diff
@@ -10,7 +10,9 @@ export interface CompatData {
   range: string;
 }
 
-export type PackageList = Record<string, PackageData>;
+export interface PackageList {
+  [key: string]: PackageData;
```

Comment:
Hm, may we rename the `key` here to represent the actual entity what is stored? e.g.: `depName: string`